src/pyosmeta/parse_issues.py

```diff
@@ -167,6 +167,7 @@ class ReviewModel(BaseModel):
     closed_at: Optional[str] = None
     issue_link: str = None
     joss: Optional[str] = None
+    partners: Optional[list[str]] = None
```

Comment:
i believe i'll need to add partners in the export keys as well to ensure it exports to yaml. 

then it would be good to add some tests around various potential "cases" of data that could be found in an issue around the partner keys